packages/analytics/metric-cards/src/styles/variables/_utilities.scss

```diff
@@ -0,0 +1,32 @@
+// Typography
+.type-xl {
+  font-size: 22px;
+}
+.type-lg {
+  font-size: 18px;
+}
+
+.type-md {
+  font-size: 16px;
+}
+
+.type-sm {
+  font-size: 14px
+}
+
+.type-xs {
+  font-size: 12px;
+}
+
+// Colors
+.color-green-500 {
+  color: var(--green-500, #07a88d);
+}
+
+.color-red-500 {
+  color: var(--red-500, #d44324);
+}
+
+.color-grey-500 {
+  color: var(--grey-500, #6f7787);
+}
```

Comment:
Please do not utilize these classes named the same as the utility classes we're trying to not use in this repo. Please use custom class names.

Reply:
Gotcha, I had a feeling you'd have an opinion on this, one way or the other :) 

This was just me being lazy, trying to get visual parity as quickly as possible.